<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="Pradeep-PC" name="tests.Regression" tests="2" failures="0" timestamp="11 Jan 2020 04:42:14 GMT" time="0.385" errors="1">
  <testcase name="afterSuite" time="0.000" classname="tests.Regression">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at selenium.framework.WebActions.closeBrowsers(WebActions.java:56)
at tests.Regression.afterSuite(Regression.java:38)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:425)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at tests.Runner.main(Runner.java:28)
at selenium.framework.App.main(App.java:14)
]]>
    </error>
  </testcase> <!-- afterSuite -->
  <testcase name="run" time="0.385" classname="tests.Regression"/>
</testsuite> <!-- tests.Regression -->
